assets/js/config.js

```diff
@@ -1,16 +1,19 @@
 import domselect from 'dom-select'
+import DLL from './lib/doubly-linked-list'
```

Comment:
If we aren't actually using this import anywhere, can we please removed it from this line?